edm4hep.yaml

```
@@ -503,6 +503,22 @@ datatypes:
       void setCovMatrix(float value, edm4hep::Cartesian dimI, edm4hep::Cartesian dimJ) { getCovMatrix().setValue(value, dimI, dimJ); }\n
       "
 
+  edm4hep::DriftChamberDigi:
+    Description: "Digitized drift chamber hit (before tracking)"
+    Author: "EDM4hep authors"
+    Members:
+      - uint64_t cellID // ID of the sensor that created this hit
+      - int32_t type // type of the raw data hit
+      - int32_t quality // quality bit flag of the hit
+      - float time [ns] // time of the hit
+      - float eDep [GeV] // energy deposited on the hit
+      - float eDepError [GeV] // error measured on eDep
+      - edm4hep::Vector3d position [mm] // point on the sensitive wire (SW) which is closest to the simhit
+      - edm4hep::Vector3d directionSW // direction of SW
```

```suggestion
      - edm4hep::Vector3d directionSW // direction of the sensitive wire (SW)
```
? Along the sensitive wire? Whose direction?

SW is like always software, so using SW should be explicit everywhere and never assume people will know this is sensitive wire.

---

SW spelled in the comment (left `directionSW` as is) and tried to clarify the meaning of this member